BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//brave/build/cargo.gni")
+
+challenge_bypass_build_path = "$root_gen_dir/challenge_bypass_ristretto/out"
+
+challenge_bypass_lib_path = challenge_bypass_build_path
+if (is_debug) {
+  challenge_bypass_lib_path += "/debug"
+} else {
+  challenge_bypass_lib_path += "/release"
+}
+
+if (is_win) {
+  challenge_bypass_lib_path += "/libchallenge_bypass_ristretto.lib"
+} else if (is_mac)
+  challenge_bypass_lib_path += "/libchallenge_bypass_ristretto.a"
+} else {
+  challenge_bypass_lib_path += "/libchallenge_bypass_ristretto.a"
+}
+
+config("external_config") {
+  include_dirs = [ "src" ]
+  libs = [ challenge_bypass_lib_path ]
+}
+
+cargo_build("challenge_bypass_ristretto_ffi") {
+  public_configs = [ ":external_config" ]
+
+  manifest_dir = "//brave/vendor/challenge-bypass-ristretto-ffi"
```

Comment:
you should remove `//brave/vendor` from here, it will still work correctly when `rebase_path` is called on it

Reply:
👍